installer/pkg/components/server/ide/configmap.go

```diff
@@ -92,36 +92,40 @@ func configmap(ctx *common.RenderContext) ([]runtime.Object, error) {
-					OrderKey: pointer.String("05"),
-					Title:    "GoLand",
-					Type:     typeDesktop,
-					Logo:     getIdeLogoPath("golandLogo"),
-					Notes:    []string{"While in beta, when you open a workspace with GoLand you will need to use the password “gitpod”."},
-					Image:    common.ImageName(ctx.Config.Repository, ide.GoLandDesktopIdeImage, ctx.VersionManifest.Components.Workspace.DesktopIdeImages.GoLandImage.Version),
+					OrderKey:    pointer.String("05"),
+					Title:       "GoLand",
+					Type:        typeDesktop,
+					Logo:        getIdeLogoPath("golandLogo"),
+					Notes:       []string{"While in beta, when you open a workspace with GoLand you will need to use the password “gitpod”."},
+					Image:       common.ImageName(ctx.Config.Repository, ide.GoLandDesktopIdeImage, ctx.VersionManifest.Components.Workspace.DesktopIdeImages.GoLandImage.Version),
+					LatestImage: common.ImageName(ctx.Config.Repository, ide.GoLandDesktopIdeImage, "latest"),
 				},
 				pycharm: {
-					OrderKey: pointer.String("06"),
-					Title:    "PyCharm",
-					Type:     typeDesktop,
-					Logo:     getIdeLogoPath("pycharmLogo"),
-					Notes:    []string{"While in beta, when you open a workspace with PyCharm you will need to use the password “gitpod”."},
-					Image:    common.ImageName(ctx.Config.Repository, ide.PyCharmDesktopIdeImage, ctx.VersionManifest.Components.Workspace.DesktopIdeImages.PyCharmImage.Version),
+					OrderKey:    pointer.String("06"),
+					Title:       "PyCharm",
+					Type:        typeDesktop,
+					Logo:        getIdeLogoPath("pycharmLogo"),
+					Notes:       []string{"While in beta, when you open a workspace with PyCharm you will need to use the password “gitpod”."},
+					Image:       common.ImageName(ctx.Config.Repository, ide.PyCharmDesktopIdeImage, ctx.VersionManifest.Components.Workspace.DesktopIdeImages.PyCharmImage.Version),
+					LatestImage: common.ImageName(ctx.Config.Repository, ide.PyCharmDesktopIdeImage, "latest"),
 				},
 				phpstorm: {
-					OrderKey: pointer.String("07"),
-					Title:    "PhpStorm",
-					Type:     typeDesktop,
-					Logo:     getIdeLogoPath("phpstormLogo"),
-					Notes:    []string{"While in beta, when you open a workspace with PhpStorm you will need to use the password “gitpod”."},
-					Image:    common.ImageName(ctx.Config.Repository, ide.PhpStormDesktopIdeImage, ctx.VersionManifest.Components.Workspace.DesktopIdeImages.PhpStormImage.Version),
+					OrderKey:    pointer.String("07"),
+					Title:       "PhpStorm",
+					Type:        typeDesktop,
+					Logo:        getIdeLogoPath("phpstormLogo"),
+					Notes:       []string{"While in beta, when you open a workspace with PhpStorm you will need to use the password “gitpod”."},
+					Image:       common.ImageName(ctx.Config.Repository, ide.PhpStormDesktopIdeImage, ctx.VersionManifest.Components.Workspace.DesktopIdeImages.PhpStormImage.Version),
+					LatestImage: common.ImageName(ctx.Config.Repository, ide.PhpStormDesktopIdeImage, "latest"),
```

Comment:
Will those image references be "resolved" (e.g. using oci-tool in `server`) or are they passed to the IDE spec like this? 
In case of the latter, this is unlikely to behave as you expect it to. Becasue the IDE image is served by registry-facade and blobserve it's heavily cached, i.e. not invalidated over the lifetime of a registry-facade/blobserve pod.

Both components make the assumption that tags are immutable - a good assumption to make overall. We should not rely on "moving tags" such as `latest`.